axelor-human-resource/src/main/java/com/axelor/apps/hr/service/KilometricService.java

```diff
@@ -211,6 +212,8 @@ public BigDecimal computeKilometricExpense(ExpenseLine expenseLine, Employee emp
       }
     }
 
+    if (expenseLine.getKilometricTypeSelect() == ExpenseLineRepository.KILOMETRIC_TYPE_ROUND_TRIP)
+      return price.multiply(BigDecimal.valueOf(2)).setScale(2, RoundingMode.HALF_UP);
```

Comment:
Always use brackets for if block.
For the scale value, you can use  appBaseService.getAppBase().getNbDecimalDigitForUnitPrice() since this is a price.